src/coreclr/inc/corpriv.h

```diff
@@ -316,49 +316,21 @@ typedef enum {
     // This is transformed into BASED_HIGHLOW or BASED_DIR64 based on the platform
     srRelocHighLow = IMAGE_REL_BASED_HIGHLOW,
 
-    // generate a .reloc for the top 16-bits of a 32 bit number, where the
-    // bottom 16 bits are included in the next word in the .reloc table
-    srRelocHighAdj,     // Never Used
-
     // generate a token map relocation, nothing into .reloc section
     srRelocMapToken,
```

Comment:
These are now all going to be shifted in value. Do these otehr values need to be updated? Specifically given the `RelocName` name map in `pewriter.cpp`.

Reply:
Oh, you've updated it. Might it make sense to actually define them too?

Reply:
I got rid of the special enum values.